care/facility/api/serializers/patient.py

```diff
@@ -456,6 +457,10 @@ class PatientNotesSerializer(serializers.ModelSerializer):
     facility = FacilityBasicInfoSerializer(read_only=True)
     created_by_object = UserBaseMinimumSerializer(source="created_by", read_only=True)
     created_by_local_user = serializers.BooleanField(read_only=True)
+    edit_window_seconds = serializers.SerializerMethodField(read_only=True)
+
+    def get_edit_window_seconds(self, obj):
+        return PATIENT_NOTE_EDIT_WINDOW
```

Comment:
The value can change, and the frontend disables the edit button according to this value returned from the backend. This way we don't need to maintain them separately in both places.

Reply:
this could be a configuration in the frontend, why would the backend have to send it each time ?

Reply:
@vigneshhari Aparna mentioned in the original issue that this value should be easily configurable. In case we maintain two separate copies of the same values which need to match, won't we have to make 2 separate PRs just to change that one value? In the current implementation, only changing it in the backend allows the frontend to adapt accordingly without any code changes on its side and it hardly takes up a few bytes of network, didn't see any downsides. That was my initial thought. If you prefer it to be maintained separately, we can remove it from the response.

Reply:
It's not about the extra bytes, sending constant data back does not make sense from an API perspective. We can allow it in this case for ease of use, but I would strongly recommend against it even if it makes it slightly inconvenient once in a while.

Reply:
I understand, in that case let's avoid sending it. I'll hardcode it in the frontend itself in a place where we can easily configure it.